src/console_cmds.cpp

```diff
@@ -614,6 +615,118 @@ DEF_CONSOLE_CMD(ConBanList)
 	return true;
 }
 
+/**
+* Loads the ban list from the specified path.
+* @param path The path to load the ban list from.
+* @return The number of entries loaded from the file or -1 if an error occurred while loading entries.
+*/
+static int LoadBanList(const std::string& path)
+{
+	const char* const group_names[] = {
+		"bans",
+		nullptr
+	};
+
+	const auto ini = new IniFile(group_names);
```

Comment:
Don't use a heap allocation for short-lived objects like this.
```suggestion
	IniFile ini { group_names };
```
This way you won't need the `delete` at the end of the function, and the code becomes safer.